zfs/replicate/cli/click.py

```diff
@@ -14,7 +14,9 @@ class EnumChoice(click.Choice):
     def __init__(self, enum: Any) -> None:
         self.__enum = enum
 
-        choices = [x.lower() for x in enum._member_names_]  # pylint: disable=protected-access
+        choices = [
+            x.lower() for x in enum._member_names_
+        ]  # pylint: disable=protected-access
```

Comment:
This is going to conflict with the other pull request.

Reply:
Yeah, I'm thinking of playing with the args options in this pull request more before landing this.